documentation/docs/get-started/alert-templates.md

```diff
@@ -59,7 +59,7 @@ The table below lists the alert templates available in PMM, organized by technol
 
 | Area | Template name | Description | Available for | Database technology |
 | :----|:------------- | :---------- | :------------ | :------------------ |
-| MongoDB | **MongoDB down** | Detects when a MongoDB instance becomes unavailable, enabling rapid response to maintain database accessibility. | All users | MongoDB |
+| MongoDB | **MongoDB down** | Detects when a MongoDB instance becomes unavailable, enabling rapid response to maintain database accessibility. When used with a MongoDB sharded cluster, the alerts based on this template will detect the unavailability of any cluster component (configuration servers, Mongos routers, data bearing nodes and arbiters) | All users | MongoDB |
```

Comment:
When monitoring MongoDB sharded clusters, alerts using this template detect outages in any cluster component (configuration servers, Mongos routers, data-bearing nodes, and arbiters).

Reply:
@sleto-it - I've replicated this in PMM3 docs with https://github.com/percona/pmm/pull/3751

Reply:
Thanks. No need to replicate, there was a PR for v3 already (I opened two since the beginning).
I think we can close the new one. I will implement your feedback in both PRs, and we can then merge imho. Thanks

<img width="866" alt="two-prs-already" src="https://github.com/user-attachments/assets/e047a22e-d06d-4b12-8a3a-855c5e4bc30f" />